src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+                        System.out.println("Now you have " + (maxcount + 1) + " tasks in the list");
+                        System.out.println("____________________________________________________________\n");
+                        maxcount++;
+                    } else if (order.contains("deadline")) {
+                        items.add (new Deadline(order.substring(9, order.length())));
+                        Deadline dummy = new Deadline(order.substring(9, order.length()));
+                        System.out.println("____________________________________________________________");
+                        System.out.println("Got it. I've added this task:");
+                        System.out.println("[D][] " + items.get(maxcount).getDescription() + dummy.getBy());
+                        System.out.println("Now you have " + (maxcount + 1) + " tasks in the list");
+                        System.out.println("____________________________________________________________\n");
+                        maxcount++;
+                    } else if (order.contains("event")) {
+                        items.add(new Event(order.substring(6, order.length())));
+                        Event dummy = new Event(order.substring(6, order.length()));
+                        System.out.println("____________________________________________________________");
+                        System.out.println("Got it. I've added this task:");
+                        System.out.println("[E][] " + items.get(maxcount).getDescription() + dummy.getDuration());
+                        System.out.println("Now you have " + (maxcount + 1) + " tasks in the list");
+                        System.out.println("____________________________________________________________\n");
+                        maxcount++;
+                    }
+                    else if(order.contains("delete")){
+                        int index = Integer.parseInt(order.substring(7,order.length()));
+                        items.remove(index-1);
+                        System.out.println("Noted I've already delete it");
+                        maxcount--;
+                    }
+
+                    order = getMission();
+                }
+                System.out.println("Bye.Hope to see you again soon!\n");
+                System.out.println("____________________________________________________________");
+                flag = false;
+            } catch (IllegalFormatException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+    }
+
+    public static String getMission() throws IllegalFormatException{
+        String mission;
+        Scanner in = new Scanner(System.in);
+        mission = in.nextLine();
+        if(!mission.contains("delete") && !mission.contains("todo") && !mission.contains("list") && !mission.contains("bye") && !mission.contains("deadline") && !mission.contains("event") && !mission.contains("done")){
```

Comment:
There should be space bar before bracket.

Also, avoid complicated expressions.

This line probably exceeds 120 chars.